### PR TITLE
sidebar close issue resolve

### PR DESCRIPTION
**Pull request by KaleemNeslit**
## Link to Issue
Closes: #9242

## Description of Changes
- Adds FIXME widget to TODO page.
- 

## "How We Fixed It"

- Save the menuName in local-storage we were setting the menuName "deafult" that's why it was closing the sidebar on navigation .
- In mobile screen we still have the same logic .
- i have to comment the animation because of it is was sliding for bit . still have the save transaction .
-let me know if you have any comment on that

## Test Plan
- Unit tested the `FIXME()` call.
- CA (click around) tested on local and frack:
  - TODO page
  - 

## Deployment Plan

1. 

## Other Considerations

-

### CODE REVIEW

File: packages/commonwealth/client/scripts/hooks/useWindowResize.ts [hunk 1]
```diff
@@ -14,7 +15,9 @@ const useWindowResize = ({ setMenu }) => {
   );
 
   const onWindowResize = useCallback(() => {
-    setMenu({ name: 'default', isVisible: !isWindowSmallInclusive });
+    if (isMobile && window.innerWidth < 425) {
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/hooks/useWindowResize.ts:18]: Why `425`?  We have breakpoints defined in `useBrowserWindow` and `breakpoints.module.scss`. Please use those values

File: packages/commonwealth/client/scripts/views/components/component_kit/cw_sidebar_menu.tsx [hunk 2]
```diff
@@ -202,7 +203,11 @@ export const CWSidebarMenu = (props: SidebarMenuProps) => {
             label: 'Explore communities',
             iconLeft: 'compassPhosphor',
             onClick: () => {
-              setMenu({ name: 'default', isVisible: false });
+              if (isMobile && window.innerWidth < 425) {
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/component_kit/cw_sidebar_menu.tsx:206]: same for 425 and below

File: packages/commonwealth/client/styles/components/sidebar/index.scss [hunk 3]
```diff
@@ -60,7 +60,7 @@
     position: absolute;
     z-index: 70;
 
-    animation: slidein 0.2s ease-in-out;
+    // animation: slidein 0.2s ease-in-out;
```
Comment (masvelio) [inline packages/commonwealth/client/styles/components/sidebar/index.scss:63]: 1. please do not commit commented out code
2. why exactly it is commented out?

File: packages/commonwealth/client/scripts/hooks/useWindowResize.ts
Comment (mzparacha) [inline packages/commonwealth/client/scripts/hooks/useWindowResize.ts:1]: lets not modify global hooks with hardcoded breakpoints

File: packages/commonwealth/client/styles/components/sidebar/index.scss (hunk 3 above)
Comment (KaleemNeslit) [reply packages/commonwealth/client/styles/components/sidebar/index.scss:63]: The issue is that whenever we navigate to the notification or explore community section, the sliding sidebar gives the impression of closing and reopening.

Comment (mzparacha) [reply packages/commonwealth/client/styles/components/sidebar/index.scss:63]: Just tested and the sidebar remains open after navigations, but I suggest to not remove the animation. Thanks!

**Review by burtonator — APPROVED** [review]

File: packages/commonwealth/client/scripts/hooks/useWindowResize.ts
Comment (masvelio) [reply packages/commonwealth/client/scripts/hooks/useWindowResize.ts:1]: @KaleemNeslit

File: packages/commonwealth/client/styles/components/sidebar/index.scss (hunk 3 above)
Comment (KaleemNeslit) [reply packages/commonwealth/client/styles/components/sidebar/index.scss:63]: @mzparacha  if we don't remove the animation we need to change the overall logic of the side bar rendering which will be complicated . i think currently it give the same user experience as old .  looking forward for your feedback

https://github.com/user-attachments/assets/d5fa5e21-d7e8-43a0-b913-310cf80bd038

File: packages/commonwealth/client/scripts/hooks/useWindowResize.ts
Comment (KaleemNeslit) [reply packages/commonwealth/client/scripts/hooks/useWindowResize.ts:1]: @masvelio  i fixed the comments related to global hooks and breakpoints

**Review by mzparacha — APPROVED** [review]

**Pull request merged**